clang/test/CIR/Lowering/atomic-runtime.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: cir-opt %t.cir -cir-to-llvm -o %t.mlir
```

Comment:
This seems fine, but I rather we test the plain LLVM output, so we don't have to worry to update these tests if the dialect changes output. Something like: `%clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o %t.ll` instead.

Reply:
I had wanted to check the LLVM IR rather than the LLVM MLIR dialect.  But all the other tests I was familiar with check the dialect, so I followed the crowd.  I will change it to generate and check LLVM IR.